src/components/Tile/Tile.js

```diff
@@ -442,7 +442,9 @@ export class ExpandableTile extends Component {
     );
 
     const tileStyle = {
-      maxHeight: this.state.tileMaxHeight + this.state.tilePadding,
+      maxHeight: this.state.expanded
+        ? `none`
```

Comment:
tiny nitpick: would it be better to set the value to `null` rather than `'none'` to remove the inline style rule entirely when the tile is collapsed?

Reply:
@emyarod Thanks for the review! Sure, I just pushed an update.